package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "scc-state",
-  "version": "1.6.0",
+  "version": "1.7.2",
```

Comment:
No need to bump the version level after each commit. Just once when you finished adding the feature :)